lib/vanagon/component.rb

```diff
@@ -326,6 +327,9 @@ def get_sources(workdir) # rubocop:disable Metrics/AbcSize
         )
         src.fetch
         src.verify
+        if source.erb
```

Comment:
You're adding a new option that can be passed with `add_source` but you didn't document it

Reply:
Does the doc at https://github.com/puppetlabs/vanagon/pull/543/files#diff-b302efb59b1963d379e36b80fb9890a5R371 suffice? or are there other place I should document it?